datajoint/schemas.py

```diff
@@ -288,6 +288,17 @@ def repl(s):
             with open(python_filename, 'wt') as f:
                 f.write(python_code)
 
+    def list_tables(self):
+        """
+        Return a list of all tables in the schema except tables with ~ in first character such
+        as ~logs and ~job
+        :return: A list of table names in their raw datajoint naming convection form
+        """
+
+        return [table_name for (table_name,) in self.connection.query("""
+            SELECT table_name FROM information_schema.tables
```

Comment:
While selecting from the `information_schema` is a MySQL-specific command, the more general and simpler SQL command for getting the table list is `SHOW TABLES`. Unless absolutely necessary, please use the more general SQL.

Reply:
@dimitri-yatsenko Wait... we should be careful here. Is it not the other way around? When using `SHOW TABLES` on postgres the following is the result:
```
postgres=# SHOW TABLES;
ERROR:  unrecognized configuration parameter "tables"
```
As opposed to how it is has been implemented here which properly returns a list of tables.

Reply:
Good point. Then we need to correct it in the other places. I had assumed that `information_schema` was MySQL specific. We have other places in the code using "SHOW TABLES".  Since we are adding this as the verified way to do it, let's also refactor the other places to call `list_tables` instead of issuing their own query.

Reply:
@dimitri-yatsenko I agree that we should make those updates, though I think it best to treat as a separate issue/PR so we don't block adding in this simple feature for use in DJ GUI. Method signature won't change except perhaps in adding more options/args which can easily be done in a backward-compatible way.